lib/src/value/calculation.dart

```diff
@@ -120,6 +122,151 @@ class SassCalculation extends Value {
+
+  /// Creates a `sin()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value atan(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("atan", [argument]);
+    }
+    return number_lib.atan(argument);
+  }
+
+  /// Creates a `asin()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value asin(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("asin", [argument]);
+    }
+    return number_lib.asin(argument);
+  }
+
+  /// Creates a `acos()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value acos(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("acos", [argument]);
+    }
+    return number_lib.acos(argument);
+  }
+
+  /// Creates a `abs()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value abs(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber) {
+      return SassCalculation._("abs", [argument]);
+    }
+    if (argument.hasUnit("%")) {
+      warnForDeprecation(
+          "abs() will not longer support percentage units."
+          "future release.\n"
```

Comment:
I think this is missing a phrase. We should probably also mention what to migrate to. `math.abs($argument)` will maintain the current behavior and `abs(#{$argument})` will make this work as a CSS function even before the deprecation is complete.

Reply:
Done! :)

Reply:
The phrasing here is still off, since it's not really that it won't support percentage units, but that the behavior will change. The documentation link should be to the breaking change page (which doesn't exist yet, but we'll add one before releasing this)

Something like this might be better:

Passing percentage units to the global abs() function is deprecated. In the future, this will emit a CSS abs() function to be resolved by the browser.

To preserve current behavior:
math.abs($argument)

To emit a CSS abs() now:
abs(#{$argument})
More info: https://sass-lang.com/d/abs-percent